scripts/enforce_licence.py

```diff
@@ -281,13 +281,16 @@ def CheckLicenceForFile(file_name, licence, lang, diff, fix):
   f = open(file_name)
   # + 1 to include the newline to have a complete line
   header_size = len(licence) + 1
+  file_name_line_count = licence.count('\n') + 2
```

Comment:
Is this not just:
```suggestion
  file_name_line_count = header_size + 1
```

Reply:
`header_size` is the total character count of the license, while `licence.count('\n')` finds specifically how many lines it consumes. This is then used to find the line number where the filename line should be and create an annotation on that line.

Reply:
Ah yeah, makes sense